interface-elements-for-web/articles/gantt/data-editing/modify-tasks.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Add, Edit and Delete Tasks
+author: Anna Kondratova
+---
+# Add, Edit and Delete Tasks
+
+## Add a Task
+
+Right-click a task and select the "Add" [context menu item](~/interface-elements-for-web/articles/gantt/context-menu.md) or click the "Add Task" [toolbar item](~/interface-elements-for-web/articles/gantt/toolbar.md) to create a task:
+
+- "New Task" - Appends a new task at the same level as the selected task.
+
+    ![](~/interface-elements-for-web/images/Gantt/add-new-task.png)
+
+- "New Subtask" - Adds a child task to the selected task.
+
+    ![](~/interface-elements-for-web/images/Gantt/add-new-subtask.png)
+
+## Edit a Task
+
+Use one of the approaches described below to edit a task.
+
+* Double-click a task in the task list. 
+
+    ![](~/interface-elements-for-web/images/Gantt/inline-editing.png)
+
+*  Right-click a task and select the "Task Details" context menu item (or double-click a task in the chart area).
+
+    ![](~/interface-elements-for-web/images/Gantt/popup-edit-form.png)
+
+* Hover the mouse pointer over the task you want to modify.
+
+    ![](~/interface-elements-for-web/images/Gantt/drag-and-drop.png)
+
+    **Move** - Hold on the task and drag it to change its start and end time.
```

Comment:
I'm not sure what you mean by "Hold on the task". Do you mean:

"Drag the edge of a task to change its start and end time."

If not, please clarify.

Reply:
What you describe is resizing - change start **or** end time.

Moving is changing both start **and** end times.

Hold is to move the mouse pointer over the task and press on it.

![image](https://user-images.githubusercontent.com/27012218/82540850-35c8c980-9b58-11ea-9caa-fa9cce7a8be1.png)

Reply:
Thank you for the detailed explanation. Can we say something like:

**Move** - Drag the task's left boundary to change its start time. Drag the right boundary to change its end time.

Reply:
The first sentence is better for me, thank you. Applied.